visualization/pg_joins.js

```diff
@@ -52,40 +52,67 @@ const fetch_specieId= async (client, specie)=>{
   }
 }
 
+const getGeometry= async (client, geometry)=>{
+  try{
+    geometry.coordinates[0].map((data)=>{
+      if(data.length<3)
+        data.push(115);
+      return data
+    })
+    const res= (await client.query('SELECT ST_Geomfromtext(ST_AsText(ST_GeomFromGeoJSON($1)),4326) As wkt', [geometry]));
```

Comment:
Nice use of parameters for SQL statements!

Reply:
Thank you!